docs/execution_patterns/remote/README.md

```diff
@@ -0,0 +1,93 @@
+# Remote Execution
+This folder provides guidance in how to use `splunk-ansible` from a controller node, setting up and provisioning a Splunk Enterprise indexer cluster on a series of remote instances. Or visually, the example shown here can be represented by the diagram below:
+
+![diagram](./remote-diagram.png)
+
+## Configure hosts
+First, we'll need a few hosts to run these Ansible plays against. For convenience, we can use a set of containers to mimic a baremetal hosts running `sshd`. These hosts are also setup to pre-install the `splunk` user. For more information, please see the [Dockerfile](./Dockerfile) in used.
+
+Additionally, see the [docker-compose.yml](./docker-compose.yml) file included for how the multiple containers are defined to assume the roles in the Splunk deployment.
+```bash
+$ docker-compose up -d
+```
```

Comment:
I'm not sure if you want to specify it like this.  The way it's written makes it seem more like an example.  Especially when interacting with the swarm you would probably use "docker stack" to deploy rather than docker-compose (which iirc works only on single hosts)

Reply:
My intention was to make this an example, but I can adjust the wording to be more clear. Basically this use case is meant to show how to use `splunk-ansible` from a control node, but IMO it's easier to walk through a more "interactive" demo using containers as target hots instead of showing a static IPs that a user may not be able to repeat.